app/src/server/api/routers/billing.router.ts

```diff
@@ -0,0 +1,147 @@
+import { sql } from "kysely";
+import { z } from "zod";
+import { createTRPCRouter, protectedProcedure } from "~/server/api/trpc";
+import { kysely, prisma } from "~/server/db";
+import { v4 as uuidv4 } from "uuid";
+import { requireCanViewProject } from "~/utils/accessControl";
+import { toUTC } from "~/utils/dayjs";
+import { success } from "~/utils/errorHandling/standardResponses";
+import { getStats } from "./usage.router";
+
+export const billingRouter = createTRPCRouter({
+  invoices: protectedProcedure
```

Comment:
I would follow our current naming convention and rename this router to be invoices.router.ts. Then I would rename `invoices` to `list`, `invoice` to `get`, `createInvoice` to `create`, etc.

Reply:
Got it, I will follow this convention!